providers/azureblob/src/main/java/org/jclouds/azureblob/blobstore/AzureBlobStore.java

```diff
@@ -368,7 +368,7 @@ public void setContainerAccess(String container, ContainerAccess access) {
 
    @Override
    public BlobAccess getBlobAccess(String container, String key) {
-      throw new UnsupportedOperationException("unsupported in Azure");
+      return BlobAccess.PRIVATE;
```

Comment:
Could you make a similar change to `RegionScopedSwiftBlobStore.getBlobAccess` and `GlacierBlobStore.getBlobAccess` for consistency?